functionary/ui/views/workflows/parameters/edit.py

```diff
@@ -41,13 +42,18 @@ def test_func(self):
         )
 
 
-class WorkflowParameterCreateView(WorkflowParameterFormViewMixin, CreateView):
+class WorkflowParameterCreateView(
+    WorkflowParameterFormViewMixin, PermissionedCreateView
+):
     """Create view for the WorkflowParameter model"""
 
     required_permission = Permission.WORKFLOW_CREATE
 
 
-class WorkflowParameterUpdateView(WorkflowParameterFormViewMixin, UpdateView):
+class WorkflowParameterUpdateView(
+    WorkflowParameterFormViewMixin, PermissionedUpdateView
+):
     """Update view for the WorkflowParameter model"""
 
     required_permission = Permission.WORKFLOW_UPDATE
+    fields = ["name", "description", "parameter_type", "default", "required"]
```

Comment:
Not really related to this PR, but the workflow field shouldn't be editable after creation, so I fixed that here.